10up/brightcove-video-connect
# Move playlist notice  and add check for other notices

### Description of the Change
This PR moves the playlist notice in the admin section outside the controls bar and adds an additional check for notices to show them in respective places



Closes #346 

### How to test the Change

- Go to the Brightcove Playlist settings page
- Observe that the notice is now outside the control bar
- Add/Edit Post
- Add Brightcove media.
- Go to a different section such as Playlist or playlist experience
- Observe that the notice is being displayed there


### Changelog Entry

> Fixed - Move playlist notice above controls bar

### Credits

Props @MARQAS , @felipeelia


### Checklist:
